cpp/src/arrow/flight/flight_test.cc

```diff
@@ -1023,7 +1023,11 @@ TEST_F(TestFlightClient, TimeoutFires) {
   Status status = client->GetFlightInfo(options, FlightDescriptor{}).status();
   auto end = std::chrono::system_clock::now();
 #ifdef ARROW_WITH_TIMING_TESTS
+#ifdef __APPLE__
+  EXPECT_LE(end - start, std::chrono::milliseconds{1000});
+#else
   EXPECT_LE(end - start, std::chrono::milliseconds{400});
+#endif
```

Comment:
Maybe just raise the timeout universally?

Reply:
done